api/health/service.md

```diff
@@ -90,125 +92,215 @@ This API uses the `json 2.0` RPC format. For more information on making JSON RPC
 
 #### `health.health`
 
-The node runs a set of health checks every 30 seconds, including a health check for each chain. This
-method returns the last set of health check results.
+This method returns the last set of health check results.
 
-**Signature:**
+**Example Call:**
 
 ```sh
-health.health() -> {
-    checks: []{
-        checkName: {
-            message: JSON,
-            error: JSON,
-            timestamp: string,
-            duration: int,
-            contiguousFailures: int,
-            timeOfFirstFailure: int
-        }
```

Comment:
I felt that this didn't add anything useful. If we wanted to we could include the golang structs defining the inputs and outputs... But I think the example shows it all.